src/values/acceleration-structure.cxx

```diff
@@ -0,0 +1,2235 @@
+/* © SPIRV-Interpreter @ https://github.com/mmoult/SPIRV-Interpreter
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+module;
+#include <algorithm>
+#include <array>
+#include <cmath>
+#include <limits>
+#include <memory>
+#include <stack>
+#include <string>
+#include <tuple>
+
+// TODO: plan to remove/change header(s) below
+#include <iostream>
+
+// TODO: probably want to reduce GLM files to link
```

Comment:
Definitely looks like a compile (not a link-time) dependency to me. Please remove this comment or explain what you meant.

Reply:
I was thinking that reducing the dependencies would reduce compile time. I guess "link" was the wrong word? But if there are no issues with including unused GLM files, I'll just remove the comment.